spec/lib/openregister_spec.rb

```diff
@@ -607,6 +610,15 @@ def stub_tsv_request url, fixture, headers: {}
           expect(display[1]).to eq "GARSTON ENTERPRISE ACADEMY 2009-02-09 - 2009-12-21"
         end
       end
+
+      describe 'retrieve a register\'s entries first page only via #_entries' do
+        it 'returns entries as Ruby objects' do
+          entries = OpenRegister.registers[1]._entries
+          expect(entries).to be_an(Array)
+          entries.each { |e| expect(e).to be_an(OpenRegister::Country) }
```

Comment:
Why would we use all, I think the test failure in this case would be better because it would explain what part of the file is wrong if it is a file issue.

if you use all you have to add an extra assertion, plus [].all? also will assert to true. 

Could be right, just wondering?